lib/l10n/app_fi.arb

```diff
@@ -0,0 +1,3 @@
+{
+  "notYetApproved": "Laitetta ei ole vielä otettu käyttöön."
```

Comment:
Not the same as app_en.arb?

Reply:
English localization is considered as the default localization and according to Flutter documentation, the description (missing in this file but present in app_en.arb) is included in the default locale.